tests/core/data/test_csv_data_node.py

```diff
@@ -102,7 +103,8 @@ def test_new_csv_data_node_with_existing_file_is_ready_for_reading(self):
         ],
     )
     def test_create_with_default_data(self, properties, exists):
-        dn = CSVDataNode("foo", Scope.SCENARIO, DataNodeId("dn_id"), properties=properties)
+        dn = CSVDataNode("foo", Scope.SCENARIO, DataNodeId(f"dn_id_{uuid.uuid4()}"), properties=properties)
```

Comment:
What s the rational for this unique id?

Reply:
To avoid duplicated id. I added this during testing and it's probably not needed anymore. But I think it's a good addition to the test

Reply:
Make sense. Thx.